citrination_client/client.py

```diff
@@ -271,19 +271,78 @@ def upload_file(self, file_path, data_set_id, root_path=None):
                 message = {"message": "Upload failed.",
                                    "status": r.status_code}
                 return json.dumps(message)
+        else:
+            return json.dumps({
+                    "message": "No file at specified path {0}".format(source_path)
+                })
+
+    def upload_file(self, file_path, dataset_id, root_path=None):
+        """
+        Upload file to Citrination.
+        :param file_path: File path to upload.
+        :param data_set_id: The dataset id to upload the file to.
+        :return: Response object or return code if the file was not uploaded.
+        """
+        return self.upload(dataset_id, file_path)
+
+    def list(self, dataset_id, glob=".", is_dir=False):
```

Comment:
Is `glob` something that the average person understands? What about something like `match`?